### PR TITLE
Apply sort settings immediately - even when in edit mode.

### LINKED ISSUE #7103: [Time List View] Time List View not sorting based on saved properties
#### Summary
Setting the Time List view to sort by Earliest First doesn't apply that setting to the view by default.

#### Expected vs Current Behavior
Setting the Time List view sort order should apply that setting to the view by default.

#### Steps to Reproduce

1. 
2.
3.
4.

#### Environment

* Open MCT Version: 
* Deployment Type: 
* OS:
* Browser:

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by shefalijoshi**
Closes #7103 

### Describe your changes:
Apply sorting immediately even when in edit mode.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: src/plugins/timelist/TimelistComponent.vue [hunk 1]
```diff
@@ -194,6 +194,7 @@ export default {
       if (this.isEditing) {
         this.filterValue = configuration.filter;
         this.hideAll = false;
+        this.setSort();
```
Comment (ozyx) [inline src/plugins/timelist/TimelistComponent.vue:197]: We should add a regression test for this

**Review by ozyx — CHANGES_REQUESTED** [review]

File: src/plugins/timelist/TimelistComponent.vue (hunk 1 above)
Comment (shefalijoshi) [reply src/plugins/timelist/TimelistComponent.vue:197]: Added a unit test for this.

**Review by ozyx — CHANGES_REQUESTED** [review]
Test is failing

**Comment by shefalijoshi** [comment]
Closing this PR. It looks like the issue is not reproducible in view-only mode actually.

**Pull request closed without merging**